aikau/src/main/resources/alfresco/lists/AlfList.js

```diff
@@ -350,6 +362,16 @@ define(["dojo/_base/declare",
       requestInProgress: false,
 
       /**
+       * An internally used attribute used to indicate whether or not a request is pending to be performed
+       * as soon as the current request completes.
+       * 
+       * @instance
+       * @type {boolean}
+       * @default
+       */
```

Comment:
Note - there's no @since statement here - but this is because it was previously present, but just not declared as an attribute.